src/Microsoft.TestPlatform.Common/Resources/Resources.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="AccessDenied" xml:space="preserve">
+    <value>{0} Access denied while trying to create "TestResults" folder in mentioned location. You are getting this exception because you are running vstest.console.exe from a folder which requires having write access. To solve the issue : please  run vstest.console.exe from a folder where you have write privileges to that folder. For more information, please look at the error message :</value>
```

Comment:
nit: The colon sign (`:`) should be right after the word (i.e. "To solve the issue: [...]", "please look at the error message: [...]").

Also, this "To solve the issue : please  run vstest.console.exe from a folder where you have write privileges to that folder." can be rephrased like this: "To solve the issue. please run vstest.console.exe from a folder where you have write privileges."

Reply:
Done.